src/profile-logic/marker-data.js

```diff
@@ -1267,6 +1267,9 @@ export function getMarkerFullDescription(marker: Marker) {
       case 'Text':
         description += ` — ${data.name}`;
         break;
+      case 'Log':
```

Comment:
Do we have existing test coverage for the marker table? It would be good to make sure we generate something that exercises this code, at least in a snapshot test.

Reply:
sadly I don't think we have any test around this yet :/